code/modules/mob/living/carbon/human/species_types/vampire.dm

```diff
@@ -44,21 +51,21 @@
 
 /datum/species/vampire/spec_life(mob/living/carbon/human/C, delta_time, times_fired)
 	. = ..()
-	if(istype(C.loc, /obj/structure/closet/crate/coffin))
-		C.heal_overall_damage(2 * delta_time, 2 * delta_time, 0, BODYPART_ORGANIC)
-		C.adjustToxLoss(-2 * delta_time)
-		C.adjustOxyLoss(-2 * delta_time)
-		C.adjustCloneLoss(-2 * delta_time)
+	if(istype(C.loc, /obj/structure/closet)) // SKYRAT EDIT - NORMAL CLOSETS INSTEAD OF COFFINS.
```

Comment:
Bodybags are just too good, sorry not sorry.
```suggestion
	if(istype(C.loc, /obj/structure/closet) && !istype(C.loc, /obj/structure/closet/body_bag)) // SKYRAT EDIT - NORMAL CLOSETS INSTEAD OF COFFINS.
```